### PR TITLE
Consolidate MapMode Gdi32 interop

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Cleanup `GetMapMode`
- Cleanup `SetMapMode`

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2137)

### CODE REVIEW

File: src/Common/src/Interop/Gdi32/Interop.MM.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class Gdi32
+    {
+        public enum MM : int
+        {
+            TEXT = 1,
```
Comment (zsd4yr) [inline src/Common/src/Interop/Gdi32/Interop.MM.cs]: should these be hex to match the _style_ of others?

**Comment by zsd4yr** [comment]
@msftbot please merge if @russkie approves

File: src/Common/src/Interop/Gdi32/Interop.MM.cs (hunk 1 above)
Comment (weltkante) [reply src/Common/src/Interop/Gdi32/Interop.MM.cs]: wasn't it the convention to match the style used in the header, or am I outdated on the conventions again? the headers use decimals for the MM constants

Comment (zsd4yr) [reply src/Common/src/Interop/Gdi32/Interop.MM.cs]: ^ ah sorry, I was seeing that the header used hex :( I may be looking at the wrong header again!

**Pull request merged**